src/main/java/sk/freemap/gpxAnimator/Renderer.java

```diff
@@ -532,6 +532,29 @@ private Point2D drawMarker(final BufferedImage bi, final int frame) {
 		return point;
 	}
 
+	private void drawSimpleCircleOnGraphics2D(Point2D point, Graphics2D g2) {
+
+		final double markerSize = cfg.getMarkerSize();
+
+		final Ellipse2D.Double marker = new Ellipse2D.Double(
+				point.getX() - markerSize / 2.0,
+				point.getY() - markerSize / 2.0,
+				markerSize,
+				markerSize);
+		g2.setStroke(new BasicStroke(1f));
+		g2.fill(marker);
+		g2.setColor(Color.black);
+		g2.draw(marker);
+	}
+
+	private void drawIconOnGraphics2D(Graphics2D g2, Point2D point) throws IOException {
```

Comment:
Again, `final` would be really nice. That is not mandatory and not a bug, but we like to see this to make sure, the parameters don't get new values assigned in the body of the method.